pkgs/test_api/lib/src/frontend/expect.dart

```diff
@@ -154,7 +154,7 @@ Future _expect(actual, matcher,
 /// Convenience method for throwing a new [TestFailure] with the provided
 /// [message].
 @alwaysThrows
```

Comment:
We can actually remove this annotation I think now as well, `Never` effectively handles the same use case.

Reply:
Done